packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -463,8 +499,38 @@ export default class DropdownCore extends React.Component<
         }
     }
 
+    renderSearchTextInput() {
+        const {items, handleSearchTextChanged} = this.props;
+        const noResult = handleSearchTextChanged && items.length === 0;
+        // TODO(jangmi): Use TextField once we have it.
```

Comment:
As of today, the `TextField` component won't include icons or any other variants, so I'd recommend for now to abstract the contents of this function into a different component and style it based off the design spec you are using for reference.

We can revisit later when building the WB TextField component.